src/seedu/addressbook/commands/Command.java

```diff
@@ -36,6 +36,10 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
         return String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, personsDisplayed.size());
     }
 
+    public static String getMessageForSortedPersonListShownSummary(List<? extends ReadOnlyPerson> personsDisplayed) {
```

Comment:
Would be good to have the header comments for the function to improve code readability. The header comments will consist of what the function would do, and what are the parameters it takes in. Which is similar to the function above this.